types/topo_paths.go

```diff
@@ -29,9 +31,11 @@ var clabTmpDir = filepath.Join(os.TempDir(), ".clab")
 // TopoPaths creates all the required absolute paths and filenames for a topology.
 // generally all these paths are deduced from two main paths. The topology file path and the lab dir path.
 type TopoPaths struct {
-	topoFile string
-	labDir   string
-	topoName string
+	topoFile           string
+	labDir             string
+	topoName           string
+	explicitCaCertFile string // if an external CA certificate is used the name to the Cert file is stored here
+	explicitCaKeyFile  string // if an external CA certificate is used the name to the Key file is stored here
```

Comment:
based on the description these fields should rather be named `externalCa...`

Reply:
and is it just a name that is stored there or a path? The comment might need to be addressed

Reply:
well there was still the discussion I tried to start in https://github.com/srl-labs/containerlab/pull/1307#issuecomment-1489906045.
so if the ca-key and ca-cert are explicitly set but they do not exist, we throw an error right now.
The other option would be to create CA key / cert under the defined, explicitly set path...
That was basically the thought process behind the name.
So two decisions to make.
1. keep the actual implementation, throwing the error or generating the CA under the specified location.
2. under the consideration in 1. rename or not

Reply:
discussed offline, the paths to ext CA and Key should be provided and be valid